packages/input-option/src/InputOption/InputOption.tsx

```diff
@@ -60,12 +84,18 @@ export const InputOption = Polymorphic<InputOptionProps, 'div'>(
         aria-checked={checked}
         tabIndex={-1}
         className={cx(
+          getContextStyles(renderedContext, state, theme),
+          getTextStyles(renderedContext, state, theme),
           {
-            [inputOptionWedge]: showWedge,
+            [inputOptionWedge]: showWedgeProp,
+            [getWedgeStyles(renderedContext, state, theme)]: shouldRenderWedge,
```

Comment:
Why are these separate?

Reply:
Not sure, but these styles have different logic for how they're rendered, and I was just maintaining that. I really just wanted to propose an alternate organizational approach to Brooke's PR